modules/profiles/hm-profiles.php

```diff
@@ -109,4 +109,14 @@ public static function loadLegacy($hmod) {
             }
         }
     }
+
+    public static function search($fld, $value) {
```

Comment:
Make as much effort as possible to avoid abbreviations in code because such code is harder to read and even debug.

Reply:
`$fld is $field` I added a little documentation to avoid confusion. Please Review

Reply:
Adding documentation is good but not what I was asking for. Please use full names.